From eaab55bb6d48643163eebbc9ca575a9ca2a8e03f Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 21 Feb 2017 17:10:07 +0200
Subject: [PATCH] configure.ac: fix sed command

The "sed 's/.bak//g'" matchs "bitbake", which would cause strange errors
when the S contains "bitbake", fix to "sed 's/\.bak$//'`"

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6907b21..7c70d9e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -948,7 +948,7 @@ YEAR=`date +%Y`
 AC_SUBST([YEAR], $YEAR)
 
 for i in ${srcdir}/src/*-args.c.bak ${srcdir}/src/*-args.h.bak; do
-	nam=$(basename $i|sed 's/.bak//g')
+	nam=$(basename $i|sed 's/\.bak$//')
 	if test "$create_libopts_links" = "yes";then
 		rm -f "src/$nam"
 		AC_CONFIG_LINKS([src/$nam:$i])
-- 
2.11.0

